my-react-app/src/app/screens/Login/components/Fields/customInput.js

```diff
@@ -0,0 +1,12 @@
+import React, { Fragment } from 'react';
+
+export const CustomInput = props => {
+  const { label, input, meta } = props;
```

Comment:
When you destructure an object, you should then use the destructured properties directly. 
Instead of `<label> {props.label} </label>`, do `<label> {label} </label>`